### ISSUE TITLE
Add some screenshots to manual

### ISSUE THREAD

**Issue by Zayon** [elmar-hinz/TX.ehfaq#2]
Hi,

Would it be possible to have some screenshots in the exentions manual ?

Thanks.

**Comment by elmar-hinz** [comment]
Yes, images are important and it the next thing to do for the extension.

If by chance you can provide useful screenshots, feel free to send them in.
